### ISSUE TITLE
[Security]:  log4j security issues

### ISSUE THREAD

**Issue by Yanx0202** [milvus-io/milvus#15265]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:v2.0.0-PreGA
- Deployment mode(standalone or cluster):cluster
- SDK version(e.g. pymilvus v2.0.0rc2): pymilvus v2.0.0rc4
- OS(Ubuntu or CentOS): windows
- CPU/Memory: 16core ,64g
- GPU: N/A
- Others:
```

### Current Behavior

pulsar v2.7.3 is using log4j-core-2.10.0.jar ,that can cause security issues.
Ref: https://logging.apache.org/log4j/2.x/security.html
       https://pulsar.apache.org/blog/2021/12/11/Log4j-CVE/

### Expected Behavior

upgrade pulsar version to 2.7.4 to fix this issues
ref: https://github.com/apache/pulsar/releases/tag/v2.7.4

### Steps To Reproduce

```markdown
see ref https://pulsar.apache.org/blog/2021/12/11/Log4j-CVE/
```

**Comment by xiaofan-luan** [comment]
are we gonna to do the upgrade?

**Comment by xiaofan-luan** [comment]
@Yanx0202 thanks for the feedback, I thought we need to do the upgrade before GA

**Comment by Yanx0202** [comment]
yes, I just commit the PR 
https://github.com/milvus-io/milvus/pull/15263

**Comment by Yanx0202** [comment]
But I am the first time to commt PR to milvus-io/milvus , So I trying to fellow the commit step

**Comment by xiaofan-luan** [comment]
> #15263

cool，the DCO seems to be not working, please check https://github.com/milvus-io/milvus/blob/master/CONTRIBUTING.md#developer-certificate-of-origin-dco to fix the DCO issues.

**Comment by xiaofan-luan** [comment]
any trouble on DCO? you probably need to check your github eamil

**Comment by xiaofan-luan** [comment]
2.7.4 seems to hit some stability issue, we are looking into it

**Comment by Yanx0202** [comment]
@xiaofan-luan ok,  is so glad to see  2.0 GA is released.

**Comment by xiaofan-luan** [comment]
> @xiaofan-luan ok, so glad to see 2.0 GA is released.

Hooray!  if you can update your pr by 2.8.2 that would be great~ 
Feel free to give us comments

**Comment by LoveEachDay** [comment]
Fix in PR #15588
